### PR TITLE
feature/add-echo-option-to-turn-on-of-in-beta-engineering-panel

### PR DESCRIPTION
**Pull request by marshalllele**
Adds On/Off checkbox to JED page

### CODE REVIEW

File: src/web/command_control/client/components/Settings.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Saving and loading settings from browser's localStorage
-import { ConstantHeadingParameters, DiveParameters, DriftParameters, Speeds } from "./shared/JAIAProtobuf"
```
Comment (michael-jaia) [inline src/web/command_control/client/components/Settings.ts]: Change the branch name to match the name on Jira (feature/add-echo-opt...)

File: src/web/jed/style.css [hunk 2]
```diff
@@ -419,3 +419,69 @@ input.manualOrPidButton:hover {
   border-width: 3pt;
   border-color: red;
 }
+
+/* JED Task Settings */
```
Comment (michael-twomey) [inline src/web/jed/style.css:423]: What is all this CSS used for? In JED, I just see a simple checkbox

File: src/web/jed/index.html [hunk 3]
```diff
@@ -204,8 +204,21 @@ <h3>Rudder</h3>
       <image src="helpPane.png">
     </div>
 
+    <div>
+      <h2>Waypoint Task Settings</h3>
```
Comment (michael-jaia) [inline src/web/jed/index.html]: What do you think about "Waypoint Settings"...seems a little crisper to me

File: src/web/jed/style.css (hunk 2 above)
Comment (michael-jaia) [inline src/web/jed/style.css]: What is all this CSS used for? In JED, I just see a simple checkbox

**Review by michael-jaia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**